src/emulator/functionsEmulator.ts

```diff
@@ -82,12 +82,12 @@ export class FunctionsEmulator implements EmulatorInstance {
     const hub = express();
 
     hub.use((req, res, next) => {
-      let data = "";
-      req.on("data", (chunk: any) => {
-        data += chunk;
+      const chunks: Buffer[] = [];
+      req.on("data", (chunk: Buffer) => {
+        chunks.push(chunk);
       });
       req.on("end", () => {
-        (req as RequestWithRawBody).rawBody = data;
+        (req as RequestWithRawBody).rawBody = Buffer.concat(chunks);
```

Comment:
Does this change `rawBody` from a `string` to `Buffer`?  Is that OK?

Reply:
Yeah it does and that's actually the original mistake. I checked in prod and it's always a buffer and it's explicitly defined as a buffer in [functions-framework](https://github.com/GoogleCloudPlatform/functions-framework-nodejs/blob/master/src/invoker.ts#L121)